include/af/arith.h

```diff
@@ -625,89 +645,106 @@ extern "C" {
     /**
        C Interface to divide one array by another array.
 
-       \param[out] out result of \p lhs / \p rhs.
-       \param[in] lhs first input
-       \param[in] rhs second input
-       \param[in] batch specifies if operations need to be performed in batch mode
-       \return \ref AF_SUCCESS if the execution completes properly
+       \param[out] out   \
+       \param[in]  lhs   first input
+       \param[in]  rhs   second input
+       \param[in]  batch batch mode
+       \return           \ref af_err
 
        \ingroup arith_func_div
     */
     AFAPI af_err af_div   (af_array *out, const af_array lhs, const af_array rhs, const bool batch);
 
     /**
-       C Interface to perform a less-than comparison between corresponding elements of two arrays.
+       C Interface to perform a less-than comparison between corresponding
+       elements of two arrays.
+
+       Output type is b8.
 
-       \param[out] out result of \p lhs < \p rhs; type is b8
-       \param[in] lhs first input
-       \param[in] rhs second input
-       \param[in] batch specifies if operations need to be performed in batch mode
-       \return \ref AF_SUCCESS if the execution completes properly
+       \param[out] out   <
+       \param[in]  lhs   first input
+       \param[in]  rhs   second input
+       \param[in]  batch batch mode
+       \return           \ref af_err
 
        \ingroup logic_func_lt
     */
     AFAPI af_err af_lt    (af_array *out, const af_array lhs, const af_array rhs, const bool batch);
 
     /**
-       C Interface to perform a greater-than comparison between corresponding elements of two arrays.
+       C Interface to perform a greater-than comparison between corresponding
+       elements of two arrays.
 
-       \param[out] out result of \p lhs > \p rhs; type is b8
-       \param[in] lhs first input
-       \param[in] rhs second input
-       \param[in] batch specifies if operations need to be performed in batch mode
-       \return \ref AF_SUCCESS if the execution completes properly
+       Output type is b8.
+
+       \param[out] out   >
```

Comment:
Same for the rest of the functions in this set.

Reply:
i think it was fine, but i changed it.